### PR TITLE
feat(46). Added base structure for Authorization/signIn/signUp screens

### PR DESCRIPTION
**Pull request by 0to1to0to1**

### CODE REVIEW

File: packages/mobile/src/components/Header/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import { Nav } from '../Nav';
+import { Logo } from '../Logo';
+import { Search } from '../Search';
+
+export const Header = () => {
+  return (
+    <View style={styles.wrapper}>
+      <Nav />
+      <Logo />
+      <Search/>
```
Comment (karolinayanko) [inline packages/mobile/src/components/Header/Header.tsx]: space is missing

**Review by karolinayanko — APPROVED** [review]

File: packages/mobile/src/components/Header/Header.tsx (hunk 1 above)
Comment (artbitbyte) [reply packages/mobile/src/components/Header/Header.tsx]: ok, thanks

File: packages/mobile/ios/Podfile.lock [hunk 2]
```diff
@@ -247,6 +258,10 @@ DEPENDENCIES:
   - React-RCTVibration (from `../../../node_modules/react-native/Libraries/Vibration`)
   - ReactCommon/jscallinvoker (from `../../../node_modules/react-native/ReactCommon`)
   - ReactCommon/turbomodule/core (from `../../../node_modules/react-native/ReactCommon`)
+  - "RNCMaskedView (from `../../../node_modules/@react-native-community/masked-view`)"
```
Comment (karolinayanko) [inline packages/mobile/ios/Podfile.lock:261]: Why does it wrapped with ""?

**Review by karolinayanko — APPROVED** [review]

File: packages/mobile/ios/Podfile.lock (hunk 2 above)
Comment (artbitbyte) [reply packages/mobile/ios/Podfile.lock:261]: This text is automatically generated when we install packages. So this is not my records, this records are generated.

**Review by belgithub — APPROVED** [review]

**Review by oleksiydubovykTR — APPROVED** [review]
Nice job!

**Review by dubbha — APPROVED** [review]

**Pull request merged**